web/sites/default/config/core.extension.yml

```diff
@@ -17,6 +17,7 @@ module:
   chosen_field: 0
   chosen_lib: 0
   ckeditor: 0
+  clamav: 0
```

Comment:
This should not have changed. We set ClamAV as part of a config split so that it is not enabled locally. Could you revert this file please?